### ISSUE TITLE
Scopes on relation property are failing

### ISSUE THREAD

**Issue by sescandell** [symfony2admingenerator/GeneratorBundle#58]
If we define a scope filtering to a field of a relation, it actually not works.

Sample:

```yaml
builders:
    list:
        params:
            title: post.title.list
            display: [ id, author, title, tags]
            scopes:
                "First Group":
                    "Empty":
                        default: true
                        filters: ~
                    "Amazing":
                        filters:
                            "tag.name": "Amazing"

```

**Comment by loostro** [comment]
@sescandell This is fixed in #96 and can be closed, right?

**Comment by sescandell** [comment]
It should be fixed yes
